<launch>
<!--  <node pkg="object_recognition" type="object_recognition"
	name="object_recognition" output="screen">
  </node>-->
  <node pkg="jsk_pcl_ros" type="cluster_point_indices_decomposer"
	name="object_recognition_output">
    <remap from="~input" to="/object_detection/output/cloud" />
    <remap from="~target" to="/object_detection/output/indices" />
  </node>
</launch>
